decidim-core/app/models/decidim/participatory_process.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module Decidim
+  # Interaction between a user and an organization is done via a
+  # ParticipatoryProcess. It's a unit of action from the Organization point of
+  # view that groups several features (proposals, debates...) distributed in
+  # steps that get enabled or disabled depending on which step is currently
+  # active.
+  class ParticipatoryProcess < ApplicationRecord
+    belongs_to :organization, foreign_key: "decidim_organization_id", class_name: Decidim::Organization
```

Comment:
Do you really need `foreign_key`? (My previous comment was regarding the migration)